web_responsive/static/src/less/navbar.less

```diff
@@ -4,6 +4,11 @@
 header {
     margin: 0;
     padding: 0;
+    
+    @media print {
+        visibility: hidden;
```

Comment:
Remove this one. Visibility hidden means that the element will still take the same amount of space, it simply won't show up. `display: none` is what you're going for, and I assume because it's later it's actually overriding this and removing the space

Reply:
hi, thanks for review.
updated.
please help to review again